packages/core/src/SlotMixin.js

```diff
@@ -49,7 +49,13 @@ const SlotMixinImplementation = superclass =>
     _connectSlotMixin() {
       if (!this.__isConnectedSlotMixin) {
         Object.keys(this.slots).forEach(slotName => {
-          if (!Array.from(this.children).find(el => el.slot === slotName)) {
+          // default slot means, we just check if there's child elements, otherwise check for slottable by slot-name
+          const hasSlottableFromUser =
+            slotName === ''
+              ? this.children.length > 0
```

Comment:
If there's a different named slot already it would unrightfully pass I thunk

Reply:
Ohh true

Reply:
Amended, added a test as well which failed before the update due to the edge case you pointed out 👍